pandas/core/indexes/period.py

```diff
@@ -579,14 +569,6 @@ def _get_string_slice(self, key: str):
         except KeyError as err:
             raise KeyError(key) from err
 
-    # ------------------------------------------------------------------------
-
-    def memory_usage(self, deep: bool = False) -> int:
```

Comment:
do we have a test that actually exercises PI for memory_usage?

Reply:
yes